erizo/src/erizo/media/SyncUtils.cpp

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
This looks awesome, but I'd consider adding some unit tests (can be found under `erizo/src/test/`)